handlers.py

```diff
@@ -35,7 +35,7 @@
 import util
 
 # Import source class files so their metaclasses are initialized.
-import blogger, flickr, github, instagram, mastodon, medium, tumblr, twitter, wordpress_rest
+import blogger, flickr, github, instagram, mastodon, medium, meetup, tumblr, twitter, wordpress_rest
```

Comment:
this is just for backfeed, feel free to omit it for Meetup until if/when we implement that.